packages/components/addon/components/hds/code-block/index.js

```diff
@@ -0,0 +1,151 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import { assert } from '@ember/debug';
+import { htmlSafe } from '@ember/template';
+import { guidFor } from '@ember/object/internals';
+import Prism from 'prismjs';
+
+import 'prismjs/plugins/line-numbers/prism-line-numbers';
+import 'prismjs/plugins/line-highlight/prism-line-highlight';
+
+// https://prismjs.com/plugins/autoloader/
+import 'prismjs/plugins/autoloader/prism-autoloader';
+
+// these are picked up from /public/ – https://cli.emberjs.com/release/writing-addons/#public
+Prism.plugins.autoloader.languages_path =
```

Comment:
Is still valid? Are we going to use it? (I though we said we would pre-embed at build-time only a subset of languages, and work on the lazy-loading later, but maybe I misunderstood)